releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -5,7 +5,15 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
 ### v6.0.0 - February, 02 2023
-* Updated the task to require Node.js 16. It previously required 10.
+* **Breaking change**:
+  With the update to the task to require Node.js 16. It previously required 10. The minimum agent version for running the task on self hosted agent changed. 
```

Comment:
This needs the same update as the other file.

Reply:
Updated and removed the before/after